standard/variables.md

```diff
@@ -1179,12 +1179,12 @@ The conditional operator ([§12.18](expressions.md#1218-conditional-operator)),
 
 #### 9.7.2.5 Function invocation
 
-For a variable `c` resulting from a ref-returning function invocation, `ref e1.M(e2, ...)`, its ref-safe-context is the narrowest of the following contexts:
+For a variable `c` resulting from a ref-returning method or local function `ref e1.M(e2, ...)`, or property invocation `ref e.P`, its ref-safe-context is the narrowest of the following contexts:
```

Comment:
The previous function wording encompassed indexers, the revised wording does not. Intentional?

Reply:
Except it didn't really in terms of the syntax given, which is *only* valid for local functions and methods.

Maybe the simplest option would be to change it to just "from a ref-returning function invocation" and *not* provide the syntax... or change it to "such as `ref e1.M(e2, ...)`"?

Reply:
Of the two I’d (slightly) favour the first, drop the code, as I don’t think the code adds much in this context.

Reply:
Done in a final commit, PTAL to check I haven't done anything stupid.